projects/packages/my-jetpack/_inc/state/selectors.js

```diff
@@ -143,6 +143,37 @@ const getStatsCounts = state => {
 	return state.statsCounts?.data;
 };
 
+const getUserStats = state => {
+	return state.userStats;
+};
+
+const getGuessedSiteLifecycleStatus = state => {
+	const { modules, purchases, features, plugins, isSiteConnected, isUserConnected } =
+		getUserStats( state );
+
+	// 'new' = no purchases + less than 3 features + less than 3 modules
+	if ( purchases.length === 0 && features.length < 3 && modules.length < 3 ) {
```

Comment:
One note about the features being checked here. I created a new jurassic ninja site without anything pre-downloaded (other than whatever comes with the Jetpack plugin normally) and as soon as I connected my user account, I had 17 features in my features array

![image](https://github.com/Automattic/jetpack/assets/65001528/0ed3e8ca-7a4d-4bbb-98ba-188ba1554948)

I am not 100% sure this is the behavior for all jetpack sites, but if it is, that would mean that as soon as a user connects their site, they'd automatically be labelled as "established" since they don't have a purchase yet and wouldn't get stopped with the "settling-in" condition

Reply:
Thanks for catching that; indeed, some features are enabled by default. I'll omit them in the features array!

Reply:
Hmmmm, even with that added defaults array, I still had 4 features active as soon as I connected my site

![image](https://github.com/Automattic/jetpack/assets/65001528/c3d607fc-fbba-4ec7-8751-9cc0fa4e3fc8)

(Ignore scan, that is because I purchased Protect)

I think those social features may be active by default? And I am also wondering if there are other default features added depending on the host?

Do you think it might be better to just leave the features out of these calculations? The modules and purchases may be enough if we can't confidently know what and how many features are active by default

Reply:
I think I contributed to this 😬 In the task description I used the word "features" to mean "Jetpack modules", not thinking about the wpcom_features array.

I agree with @CodeyGuyDylan - let's simplify this by dropping the features check and just use purchases and Jetpack modules.

So sorry for the mislead there @robertsreberski 🤦

Reply:
Yeah, that simplifies things 😅 sounds good, thanks for your feedback here 👍